gradoop-temporal/src/main/java/org/gradoop/temporal/model/impl/TemporalGraphCollection.java

```diff
@@ -227,14 +234,50 @@ public GraphCollection toGraphCollection() {
    * {@link GraphCollection} with default values for the temporal attributes.
    *
    * @param graphCollection the existing graph collection instance
-   * @return a temporal graph colection with default temporal values
+   * @return a temporal graph collection with default temporal values
    * @see TemporalGraphCollectionFactory#fromNonTemporalGraphCollection(BaseGraphCollection)
    */
   public static TemporalGraphCollection fromGraphCollection(GraphCollection graphCollection) {
     return TemporalGradoopConfig.fromGradoopFlinkConfig(graphCollection.getConfig())
       .getTemporalGraphCollectionFactory().fromNonTemporalGraphCollection(graphCollection);
   }
 
+  /**
+   * Convenience API function to create a {@link TemporalGraphCollection} from an existing
+   * {@link GraphCollection} with valid times depending on the 3 given {@link TimeIntervalExtractor} functions
```

Comment:
```suggestion
   * {@link GraphCollection} with valid times depending on the three given {@link TimeIntervalExtractor} instances.
```
Maybe a line break if it exceeds the max chars per line